packages/phasor/package.json

```diff
@@ -37,6 +37,7 @@
     }
   },
   "dependencies": {
+    "@blocksuite/blocks": "workspace:*",
```

Comment:
This leads to circular deps and should be reverted.

Reply:
> This leads to circular deps and should be reverted.

ok, I get it.
because I want to reuse some basic code library without repeating. 
if we want to use the Point Class in block packages, Should we re-write the Point class in phasor package or 
should we extract the some basic codes into a new package that is a very basic packages for use of other packages. Maybe just called basic or math?
Which do you perfer？